File: JavaProjects/FactoryMethodSolucao/src/boletobancario/bancodobrasil/BancoDoBrasilBoleto10Dias.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package boletobancario.bancodobrasil;

import boletobancario.Boleto;

/**
 *
 * @author jeffe
 */
public class BancoDoBrasilBoleto10Dias extends Boleto {

    public BancoDoBrasilBoleto10Dias(double valor) {
        this.valor=valor;
        this.juros = 0.03;
        this.desconto=0.05;
        this.multa=0.02;
    } 
}

File: CsharpProjects/FactoryMethodSolucao/FactoryMethodSolucao/boletobancario/Bancodobrasil/BancoDoBrasilBoleto10Dias.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using FactoryMethodSolucao.boletobancario;

namespace FactoryMethodSolucao.boletobancario.Bancodobrasil
{
    public class BancoDoBrasilBoleto10Dias : Boleto
    {
        public BancoDoBrasilBoleto10Dias(double valor)
        {
            this.valor = valor;
            juros = 0.03;
            desconto = 0.05;
            multa = 0.02;
        }

    }
}
